dissect/target/plugins/apps/webservers/webservers.py

```diff
@@ -0,0 +1,56 @@
+from dissect.target.exceptions import UnsupportedPluginError
+from dissect.target.helpers.record import TargetRecordDescriptor
+from dissect.target.plugin import Plugin, export
+
+WebserverRecord = TargetRecordDescriptor(
+    "application/log/webserver",
+    [
+        ("datetime", "ts"),
+        ("string", "remote_user"),
+        ("net.ipaddress", "ipaddr"),
+        ("uri", "file_path"),
```

Comment:
```suggestion
        ("path", "source"),
```
And move to the end of the descriptor.